src/Types/CompoundType.php

```diff
@@ -110,7 +111,7 @@ public function isSubtypeSkipped($key): bool
 	/**
 	 * @param string|int $key
 	 */
-	public function overwriteInvalidSubtype($key, Type $type): void
+	public function overwriteInvalidSubtype($key, WithTypeAndValue $exception): void
```

Comment:
`CompoundType` should probably change interface, for each subtype is value gonna be the same

Reply:
Change how? Tell me plx :D

Reply:
`setInvalidSubtypes(string|int $key, array<Type> $types, mixed $value)`
Bit inconsistent with other invalidation apis, but passing value with each Type really dost not make sense here

Reply:
So... rename `overwriteInvalidSubtype` to `setInvalidSubtypes`?

Reply:
And set them all with single method call. The $key paramter should be moved to $types probably as well, I forgot the first time